pkg/migration/registry.go

```diff
@@ -471,6 +502,24 @@ func (d *delegatingConverter) ComposedTemplate(sourceTemplate xpv1.ComposedTempl
 	return d.cmpFn(sourceTemplate, convertedTemplates...)
 }
 
+func DefaultCompositionConverter(convertTags bool, conversionMap map[string]string) ComposedTemplateConversionFn {
+	return func(sourceTemplate xpv1.ComposedTemplate, convertedTemplates ...*xpv1.ComposedTemplate) error {
+		var patchesToAdd []xpv1.Patch
+		var err error
+		if convertTags {
```

Comment:
With a reasoning similar to the one in the `DefaultPatchSetsConverter` function, would it be possible to move the tag conversion functionality to a community AWS provider specific conversion function and leave only the provider-independent fieldpath mapping in the core? The `value` and the `key` fields are specific to the `Tag` objects of the community AWS provider.

Reply:
Moved!